src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CreateClient.cs

```diff
@@ -50,6 +51,38 @@ public static void EmptyStringServerName_Throws_ArgumentException(PipeDirection
             Assert.Throws<ArgumentException>(() => new NamedPipeClientStream("", "client1", direction, PipeOptions.None, TokenImpersonationLevel.None));
         }
 
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void ReservedPipePath_Throws_ArgumentOutOfRangeException()
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(() => new NamedPipeClientStream(".", "anonymous"));
```

Comment:
The server version of this test has "anonymous" in a local const and tests other overloads of NamedPipeClientStream's ctor as well.  It'd be nice to be consistent.